contracts/Cargo.toml

```
@@ -1,11 +1,14 @@
 [workspace]
 resolver = "2"
 members = [
-  "contracts/*",
+  "*",
+]
+exclude = [
+  "target",
 ]
 
 [workspace.dependencies]
-soroban-sdk = "22.0.6"
+soroban-sdk = "23.2.1"
```

Latest version is a bit newer:
```suggestion
soroban-sdk = "23.4.0"
```